src/main/resources/htmlpublisher/HtmlPublisherTarget/BaseHTMLAction/index.groovy

```diff
@@ -26,9 +28,10 @@ def serveWrapper() {
     def footer = HtmlPublisher.class.getResourceAsStream("/htmlpublisher/HtmlPublisher/footer.html").text
 
     raw(header)
+    script(src: "${Jenkins.get().getRootUrl() + Functions.getResourcePath()}/plugin/htmlpublisher/js/htmlpublisher.js", type: "text/javascript")
 
     def legacyFile = new File(my.dir(), "htmlpublisher-wrapper.html")
-    def matcher = legacyFile.text =~ /<li id="tab\d+" class="unselected" onclick="updateBody\('tab\d+'\);" value="([^"]+)">([^<]+)<\/li>/
+    def matcher = legacyFile.text =~ /<li id="tab\d+" class="unselected"(?: onclick="updateBody\('tab\d+'\);")? value="([^"]+)">([^<]+)<\/li>/
```

Comment:
Here we need to catch the reports that were generated before the change too so that data can be extracted from them when using `useWrapperFileDirectly: false` mode. So we optionally match the inline handler in a non-capturing group.